tests/python/contrib/test_hexagon/pytest_util.py

```diff
@@ -109,25 +113,32 @@ def get_multitest_ids(
     ]
 
 
-def get_numpy_dtype_info(np_dtype_name: str) -> Union[np.finfo, np.iinfo]:
+def get_numpy_dtype_info(dtype: Union[str, np.dtype]) -> Union[np.finfo, np.iinfo]:
     """
     Return an appropriate 'np.iinfo' or 'np.finfo' object corresponding to
-    the specified dtype.
+    the specified Numpy dtype.
     """
-    np_dtype = np.dtype(np_dtype_name)
+
+    if type(dtype) == str:
```

Comment:
isinstance(dtype, str)

Reply:
Good idea.  I put the assert there because AFAIK Python `typing` typically isn't checked at runtime.  So I put the assertion there to help detect such problems.

But in retrospect, that's not particularly idiomatic.  I'll just delete the assert.

Reply:
Actually, a better solution is to drop the explicit typing, and just document that that arg needs to be something that Numpy can deal with.  I'll make that change instead.